### PR TITLE
Player hotkeys

### LINKED ISSUE #932: Add ability to mute, increase/decrease volume with keyboard

### LINKED ISSUE #934: Add ability to move video 5 and 10 seconds forward and backward with keyboard
Shortcuts here https://github.com/Joystream/atlas/issues/823

### LINKED ISSUE #936: Add ability to toggle full-screen on pressing F key

### PR DESCRIPTION
**Pull request by drillprop**
Fix #936 #934 #932 
I definetely overestimate these :P

### CODE REVIEW

**Comment by drillprop** [comment]
You can find all shortcuts here: https://github.com/Joystream/atlas/issues/823

File: src/shared/components/VideoPlayer/videoJsPlayer.ts [hunk 1]
```diff
@@ -38,14 +99,22 @@ export const useVideoJsPlayer: VideoJsPlayerHook = ({
...
   useEffect(() => {
+    if (!playerRef.current) {
+      return
+    }
     const videoJsOptions: VideoJsPlayerOptions = {
       controls: true,
       // @ts-ignore @types/video.js is outdated and doesn't provide types for some newer video.js features
       playsinline: true,
+      userActions: {
+        hotkeys: (e) => hotkeysHandler(e, playerInstance),
```
Comment (rafalpawlow) [inline src/shared/components/VideoPlayer/videoJsPlayer.ts]:
```suggestion
        hotkeys: (event) => hotkeysHandler(event, playerInstance),
```

File: src/views/viewer/VideoView/VideoView.tsx [hunk 2]
```diff
@@ -148,14 +132,12 @@ export const VideoView: React.FC = () => {
         <PlayerContainer>
           {video ? (
             <VideoPlayer
-              playing={playing}
+              autoplay={true}
```
Comment (rafalpawlow) [inline src/views/viewer/VideoView/VideoView.tsx]:
```suggestion
              autoplay
```

File: src/shared/components/VideoPlayer/videoJsPlayer.ts [hunk 3]
```diff
@@ -18,6 +18,67 @@ export type VideoJsConfig = {
   onTimeUpdated?: (time: number) => void
 }
 
+const hotkeysHandler = (e: videojs.KeyboardEvent, playerInstance: VideoJsPlayer) => {
```
Comment (rafalpawlow) [inline src/shared/components/VideoPlayer/videoJsPlayer.ts]:
```suggestion
const hotkeysHandler = (event: videojs.KeyboardEvent, playerInstance: VideoJsPlayer) => {
```

**Review by rafalpawlow — CHANGES_REQUESTED** [review]

**Review by rafalpawlow — APPROVED** [review]

**Pull request merged**